src/test/java/com/commercetools/testUtil/customTestConfigs/CtpClientConfig.java

```diff
@@ -0,0 +1,27 @@
+package com.commercetools.testUtil.customTestConfigs;
+
+import io.sphere.sdk.client.SphereClient;
+import io.sphere.sdk.client.SphereClientConfig;
+import io.sphere.sdk.client.SphereClientFactory;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+public class CtpClientConfig {
+
+    @Value("${paypalplus-integration-test.ctp.client.projectKey}")
```

Comment:
no class - no problem. though it might be incompatible with previous projects configuration, like `payone-integration-service`. Follow [the discussion](https://github.com/commercetools/commercetools-paypal-plus-integration/issues/3#issuecomment-318323604)